de.danielnoethen.butt.json

```diff
@@ -0,0 +1,108 @@
+{
+    "app-id": "de.danielnoethen.butt",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "24.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "butt",
+    "rename-icon": "butt",
+    "rename-desktop-file": "butt.desktop",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-run/pipewire-0",
+        "--device=dri",
+        "--filesystem=home",
```

Comment:
what is `home` for?

Reply:
Gives read and write access to the users home directory. 
It's used by other flatpak applications here.

Reply:
> It's used by other flatpak applications here. 

two wrongs don't make a right. "others do it" is NEVER the justification.

Reply:
> > It's used by other flatpak applications here. 
> 
> two wrongs don't make a right. "others do it" is NEVER the justification.

I didn't realise it was a wrong. I had implemented the feature here like this because I had used the other manifests as a reference guide. 

I have however looked up the spec from the flatpak documentation and using `home` for giving access to the home directory is a valid and correct way for doing this.

https://docs.flatpak.org/en/latest/sandbox-permissions.html

Reply:
> what is `home` for?

As for a use case explanation because I realise I never gave one. The application allows you to save recordings, logs, and settings all to their own files. 
These can also be loaded into the application. 
The application also maintains a default config at ~/.buttrc
Giving the home permission ensures we don't lock filesystem permissions down too much and limit usability of the application.

Reply:
>  The application also maintains a default config at ~/.buttrc

This is just wrong. It should use xdg standard directories and not drop config files in home.

Reply:
I'd agree with you, however I'm not the applications original developer and since the application heavily documents where this config file is changing it up would potentially cause confusion. 
It's not always easy to make that kind of different clear on a flatpak app.